### PR TITLE
Add minimal istio profile

### PR DESCRIPTION
**Pull request by linsun**
working on test, will be in a separate PR.

### CODE REVIEW

File: install/kubernetes/helm/istio/values-istio-minimal.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+security:
+  enabled: false
+
+sidecarInjectorWebhook:
+  enabled: false
+
+galley:
+  enabled: false
+
+mixer:
```
Comment (ymesika) [inline install/kubernetes/helm/istio/values-istio-minimal.yaml]: shouldn't this be `mixer.pilot=false, mixer.telemetry=false`?

Comment (linsun) [reply install/kubernetes/helm/istio/values-istio-minimal.yaml]: yes, my bad, cp and paste.  thanks!

**Comment by linsun** [comment]
@ymesika PTAL comments addressed

**Comment by linsun** [comment]
@ymesika tracing is disabled by default, see values.yaml.  re galley, Ozzy told me it is not required to be enabled by default in 1.1.

**Comment by ymesika** [comment]
> @ymesika tracing is disabled by default

https://github.com/istio/istio/blob/release-1.1/install/kubernetes/helm/istio/values.yaml#L230

**Comment by linsun** [comment]
@ymesika you are correct!  :)  I was basically enabling the same config nob we documented here:  https://istio.io/docs/setup/kubernetes/minimal-install/

so it was tested with enableTracing default value as no overwrite is issued for it.

**Pull request merged**